### PR TITLE
RequestResponseLogger: Reduce Log-Level to TRACE #9899

### LINKED ISSUE #9899: RequestResponseLogger: Reduce Log-Level from DEBUG to TRACE [0.5]
### Problem Description

The RequestResponseLogger class in sormas-rest is a utility class to help debugging rest calls. By default it's writing with log level debug, which means it is output into the application.debug log file by default and thus leads to unnecessary "spam" in the file system.

### Proposed Change

- [x] Reduce the log level of RequestResponseLogger from debug to trace, so that logging can still be enabled when needed, but is not active by default.

### Acceptance Criteria
* application.debug log file does not contain logs written by RequestResponseLogger  anymore.

### Additional Information
https://github.com/hzi-braunschweig/SORMAS-Security/issues/47

### PR DESCRIPTION
**Pull request by syntakker**
Fixes #9899

### CODE REVIEW

**Review by MateStrysewske — APPROVED** [review]
Looks fine by me, but I'd like Stefan to take a quick look at it as well

File: sormas-rest/src/main/java/de/symeda/sormas/rest/logging/RequestResponseLogger.java [hunk 1]
```diff
@@ -67,20 +67,19 @@ public void destroy() {
...
 	 */
 	public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-		// request logging
-		HttpServletRequest httpRequest = (HttpServletRequest) request;
-		logger.debug("requestUri = {}", httpRequest.getRequestURI());
-
-		Map<String, String[]> params = httpRequest.getParameterMap();
-		for (String s : params.keySet()) {
-			logger.debug("  {} = {}", s, httpRequest.getParameter(s));
-		}
```
Comment (StefanKock) [inline sormas-rest/src/main/java/de/symeda/sormas/rest/logging/RequestResponseLogger.java:77]: Missing empty line after method declaration

**Review by StefanKock — CHANGES_REQUESTED** [review]
Only one code formatting finding

File: sormas-rest/src/main/java/de/symeda/sormas/rest/logging/RequestResponseLogger.java (hunk 1 above)
Comment (syntakker) [reply sormas-rest/src/main/java/de/symeda/sormas/rest/logging/RequestResponseLogger.java:77]: Done

**Pull request merged**